lineage/figures/figureCommon.py

```diff
@@ -120,7 +120,7 @@ def commonAnalyze(list_of_populations, xtype="length", **kwargs):
     elif xtype == "bern":
         x = paramTrues[:, 0, 0]
 
-    return x, paramEst, dictOut["accuracy_after_switching"], dictOut["transition_matrix_norm"], dictOut["pi_vector_norm"], paramTrues
+    return x, paramEst, dictOut["accuracy_after_switching"], dictOut["transition_matrix_norm"], dictOut["pi_vector_norm"], paramTrues, dictOut["balanced_accuracy_score"]
```

Comment:
Might just be easier to replace `dictOut["accuracy_after_switching"]` with `dictOut["balanced_accuracy_score"]` and keep the number of output the same